esi_ui/api/esi_api.py

```diff
@@ -147,6 +147,23 @@ def deploy_node(request, node):
     if 'ssh_keys' in kwargs:
         kwargs['config'] = instance_config.GenericConfig(ssh_keys=kwargs['ssh_keys'])
         del kwargs['ssh_keys']
+    
+    network_id = kwargs['nics'][0]['network']
+    
+    if kwargs['floatingIPOption'] == 'none':
+        kwargs['nics'] = [{'network': network_id}]
+    else:
+        connection = esiclient(request)
+        node = connection.baremetal.get_node(node)
```

Comment:
Can you call this `ironic_node` or something to prevent the variable from being overwritten?

Reply:
done!